versions.props

```diff
@@ -57,6 +57,7 @@ net.jpountz.lz4:lz4 = 1.3.0
 one.util:streamex = 0.8.1
 org.apache.commons:commons-lang3 = 3.12.0
 org.apache.commons:commons-pool2 = 2.11.1
+org.apache.commons:commons-text = 1.10.0
```

Comment:
`./gradlew why --hash b5102f9b` shows that this is being pulled in by `dropwizard-configuration, which is only used by our ETE tests.